project/src/main/java/com/google/sps/utilities/WrapperDatabaseService.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.utilities;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+
+/** Class that initializes spanner and database client. */
+public class WrapperDatabaseService implements DatabaseService {
```

Comment:
DatabaseServiceImpl instead of WrapperDatabaseService.

Reply:
Done.